q2-structure/proposal-by-xdg.md

```diff
@@ -0,0 +1,222 @@
+Council members.
+
+In concert with the Steering Council, they manage their own membership and
+have the power to modify the Perl Rules of Governance.
+
+The Core Team do not have any authority over parts of the Perl ecosystem
+unrelated to developing and releasing the language itself.
+
+### Membership, Inactivity, and Removal
+
+The initial Core Team members will be specified in the first draft of the Perl
+Rules of Governance.
+
+The Perl Rules of Governance document will provide guidance and examples of
+desired characteristics of Core Team members.
+
+Core Team members who have stopped contributing are encouraged to declare
+themselves as "inactive". Inactive members do not participate in votes.
+Inactive members may declare themselves active at any time, except when a vote
+has been proposed and is not concluded.  Eligibility to vote will be
+determined by the Vote Administrator.
+
+Core Team Members may also resign their position at any time.
+
+Any Core Team member may nominate someone to be added to the Core Team by
+sending the nomination to the Steering Council.  The Steering Council must
+approve or reject the nomination.  If approved, the Steering Council will
+organize a Membership Change Vote to ratify the addition.
+
+In exceptional circumstances, it may be necessary to remove someone from the
+Core Team against their will, such as for flagrant or repeated violations of a
+Code of Conduct.  Any Core Team member may send a recall request to the
+Steering Council naming the individual to be removed.  The Steering Council
+must approve or reject the recall request.  If approved, the Steering Council
+will organize a Membership Change vote to ratify the removal.
+
+If the removed member is also on the Steering Council, then they are removed
+from the Steering Council as well.
+
+### No Confidence in the Steering Council
+
+The Core Team may remove either a single Steering Council member or the entire
+Steering Council via a No Confidence Vote.
+
+A No Confidence Vote is triggered when a Core Team member calls for one
+publicly on an appropriate project communication channel, and another Core
+Team member seconds the proposal.
+
+If a No Confidence Vote removes all Steering Council members, the Vote
+Administrator of the No Confidence Vote will then administer a Term Election
+to select a new Steering Council.
+
+### Amending Perl Rules of Governance
+
+Any Core Team member may propose amending the Perl Rules of Governance by
+sending a proposal to the Steering Council.  The Steering Council must decide
+to approve or reject the proposal.  If approved, the Steering Council will
+administer an Amendment Vote.
+
+## The Perl Steering Council
+
+The Perl Steering Council is a 5-person committee, elected by the Perl Core
+Team.  Candidates are not required to be members of the Core Team.  Non-member
+candidates are added to the Core Team if elected as if by a Membership Change
+Vote.
+
+### Powers
```

Comment:
I also have concerns about this section being overly specific.

1. This section is vague on if the council can delegate. If the council can't delegate then decisions are very expensive because everything has to be voted on.
2. The section is sufficiently specific that I'm left wondering if the council even has authority over the perl/perl5 repo.
3. I think it's a long term mistake to take away responsibilities from the council. The ones being taken away might one day come in scope for them in which case this has to be undone. What was the value of limiting them to begin with?

I propose replacing this section with this which is much closer to the original PEP-13 text. I find it works well for the council too.

```
The council has broad authority to make decisions about the project.
For example, they can:

- Enforce or update the project's code of conduct.
- Work with the TPF to manage any project assets.
- Delegate parts of their authority to teams or individuals.

It does not have the power to modify the Perl Rules of Governance.

Decisions of the Steering Council will be made by majority vote of non-vacant seats on the council.
```

Reply:
I've included examples (including delegation) in the revised version I just pushed.